nntrainer/include/neuralnet.h

```diff
@@ -123,23 +123,23 @@ class NeuralNetwork {
    * @param[in] input Input Tensor X
    * @retval    Output Tensor Y
    */
-  Tensor forwarding(Tensor input);
+  Tensor forwarding(Tensor input, int &status);
 
   /**
    * @brief     forward propagation
    * @param[in] input Input Tensor X
    * @param[in] label Input Tensor Y2
    * @retval    Output Tensor Y
    */
-  Tensor forwarding(Tensor input, Tensor output);
+  Tensor forwarding(Tensor input, Tensor output, int &status);
 
   /**
    * @brief     back propagation to update W & B
    * @param[in] input Input Tensor X
    * @param[in] expectedOutput Lable Tensor Y
    * @param[in] iteration Epoch Number for ADAM
    */
-  void backwarding(Tensor input, Tensor expected_output, int iteration);
+  int backwarding(Tensor input, Tensor expected_output, int iteration);
```

Comment:
IMO, the docs should be updated as well.

Reply:
This will be updated. changing for the forwarding is enough for now.